Add how to install jekyll

dr0i opened the pull request:
I had many issues by using `jekyll` from the official repositories, as well as on Debian Testing as on Debian Bookworm.
The solution here worked for me (well I hope I have written down everything correctly).

TobiasNx commented on README.md:
````diff
@@ -7,11 +7,26 @@ For the documentation see: http://metafacture.github.io/metafacture-documentatio
+## Installation
+If you are on debian derivates go install build-essentials:
+```
+ apt install build-essential
````
Is the whitespace intentional?

dr0i replied:
Yeah I thought so because here it is also one prefixing whitespace: https://github.com/metafacture/metafacture-documentation/blob/master/docs/Documentation-Maintainer-Guide.md?plain=1#L33C1-L34C2 , but at other places it is not. We can or should remove all prefixing whitespaces?

dr0i replied:
I've removed these whitespaces here.

dr0i commented:
Note: if this PR is good, we could add the jekyll installation advices to https://github.com/metafacture/metafacture-tutorial/blob/main/README.md.

Phu2 commented on README.md:
````diff
@@ -7,11 +7,26 @@ For the documentation see: http://metafacture.github.io/metafacture-documentatio
+```
+Then you can use the Ruby Dependency Management to build all you need:
+```
+ bundle install jekyll
````
This won't work, only `bundle install` is needed.

dr0i replied:
Thx! Fixed in https://github.com/metafacture/metafacture-documentation/pull/56/commits/411d0367fd7023bd9e52a34ab4c07222bb65af7b .

Phu2 commented:
> I had many issues by using jekyll from the official repositories, as well as on Debian Testing as on Debian Bookworm.

I tried to avoid installing jekyll on my system by using these docker images but no luck.
- https://hub.docker.com/r/jekyll/jekyll/ (Last updated over 2 years ago, `bundle install` failed)
- https://hub.docker.com/r/bretfisher/jekyll-serve (Up-to-date but downloading gems from rubygems.org failed, see below)

> jekyll-1  | Bundler 2.6.8 is running, but your lockfile was generated with 2.3.26. Installing Bundler 2.3.26 and restarting using that version.
> jekyll-1  | Fetching source index from https://rubygems.org/
> jekyll-1  | 
> jekyll-1  | Retrying fetcher due to error (2/6): Bundler::HTTPError Could not fetch specs from https://rubygems.org/ due to underlying error <Gem::Net::OpenTimeout: Failed to open TCP connection to rubygems.org:443 (execution expired) (https://rubygems.org/specs.4.8.gz)>
> jekyll-1  | Fetching source index from https://rubygems.org/
> jekyll-1  | 
> jekyll-1  | Retrying fetcher due to error (2/6): Bundler::HTTPError Could not fetch specs from https://rubygems.org/ due to underlying error <Gem::Net::OpenTimeout: Failed to open TCP connection to rubygems.org:443 (execution expired) (https://rubygems.org/specs.4.8.gz)>
> jekyll-1  | 
> jekyll-1  | Retrying fetcher due to error (3/6): Bundler::HTTPError Could not fetch specs from https://rubygems.org/ due to underlying error <Gem::Net::OpenTimeout: Failed to open TCP connection to rubygems.org:443 (execution expired) (https://rubygems.org/specs.4.8.gz)>
> jekyll-1  | 
> jekyll-1  | Retrying fetcher due to error (4/6): Bundler::HTTPError Could not fetch specs from https://rubygems.org/ due to underlying error <Gem::Net::OpenTimeout: Failed to open TCP connection to rubygems.org:443 (execution expired) (https://rubygems.org/specs.4.8.gz)>
> jekyll-1  | Retrying fetcher due to error (5/6): Bundler::HTTPError Could not fetch specs from https://rubygems.org/ due to underlying error <Gem::Net::OpenTimeout: Failed to open TCP connection to rubygems.org:443 (execution expired) (https://rubygems.org/specs.4.8.gz)>
> jekyll-1  | 
> jekyll-1  | 
> jekyll-1  | Retrying fetcher due to error (6/6): Bundler::HTTPError Could not fetch specs from https://rubygems.org/ due to underlying error <Gem::Net::OpenTimeout: Failed to open TCP connection to rubygems.org:443 (execution expired) (https://rubygems.org/specs.4.8.gz)>
> jekyll-1  | 
> jekyll-1  | Could not fetch specs from https://rubygems.org/ due to underlying error
> jekyll-1  | <Gem::Net::OpenTimeout: Failed to open TCP connection to rubygems.org:443
> jekyll-1  | (execution expired) (https://rubygems.org/specs.4.8.gz)>

dr0i commented:
I've added a commit that takes you suggestions into account.
Also, I have tried the installation guide on my laptop. I did not work, only after purging jekyll via packet manager - so I added this to the README.
Now it works here, I am glad :rocket:
